src/blocks/BulletPointBlock/BulletPointBlock.tsx

```diff
@@ -0,0 +1,217 @@
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from 'blocks/BaseBlockProps'
+import { FontSizes, Heading } from 'components/Heading/Heading'
+
+const BulletPointSectionWrapper = styled(SectionWrapper)`
+  overflow-x: hidden;
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+
+  ${MOBILE_BP_DOWN} {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  ${MOBILE_BP_UP} {
+    margin-left: -1.5rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    margin-left: -3.5rem;
+  }
+`
+
+const BulletPointTitle = styled(Heading)`
+  margin-bottom: 1rem;
+`
+
+const BulletPoint = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+  columns: number
+}>`
+  display: flex;
+  flex-direction: ${(props) => (props.bulletPointLayout ? 'row' : 'column')};
+  width: 100%;
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+
+  ${(props) =>
+    props.alignCenter &&
+    `
+      align-items: center;
+      text-align: center;
+    `}
+
+  ${MOBILE_BP_UP} {
+    flex-direction: column;
+    width: calc(50% - 1.5rem);
+    margin-left: 1.5rem;
+  }
+
+  ${TABLET_BP_UP} {
+    ${(props) =>
+      props.columns === 3 &&
+      `
+        width: calc((1 / 3 * 100%) - 1.5rem)
+      `}
+  }
+
+  ${LAPTOP_BP_UP} {
+    width: ${(props) => `calc((1 / ${props.columns} * 100%) - 3.5rem)`};
+    margin-left: 3.5rem;
+  }
+`
+
+const BulletPointHead = styled.div<{
+  alignCenter: boolean
+  bulletPointLayout: boolean
+}>`
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  margin-bottom: 1.5rem;
+
+  ${(props) =>
+    props.bulletPointLayout &&
+    `
+        justify-content: flex-start;
+        max-width: 2rem;
+        max-height: 2rem;
+        margin-right: 1.5rem;
+
+        ${MOBILE_BP_UP} {
+          max-width: none;
+          max-height: none;
+          width: auto;
+          margin-right: 0;
+        }
+      `}
+`
+
+const BulletPointImage = styled(DeferredImage)<{
+  alignCenter: boolean
+}>`
+  width: auto;
+`
+
+const BulletPointContent = styled.div``
+
+const BulletPointBody = styled.div<{
+  alignCenter: boolean
+  colorComponent: MinimalColorComponent
+}>`
+  max-width: ${(props) => (props.alignCenter ? '16rem' : 'none')};
+  margin-left: auto;
+  margin-right: auto;
+  color: ${(props) =>
+    getMinimalColorStyles(props.colorComponent?.color ?? 'standard').color};
+  font-size: 1rem;
+
+  & > * {
+    margin: 0 auto 1rem;
+  }
+
+  ${LAPTOP_BP_UP} {
+    font-size: 1.125rem;
+  }
+`
+
+export type BulletPointItemProps = ReadonlyArray<
+  BrandPivotBaseBlockProps & {
+    image: Image
+    title?: string
+    title_size: FontSizes
+    title_size_mobile?: FontSizes
+    paragraph: MarkdownHtmlComponent
+  }
+>
+
+export type BulletPointBlockProps = BrandPivotBaseBlockProps & {
+  align_center: boolean
+  bullet_point_layout: boolean
+  color_body: MinimalColorComponent
+  bullet_points: BulletPointItemProps
+}
+
+export const BulletPointBlock: React.FC<BulletPointBlockProps> = ({
```

Comment:
I think we said we're moving away from using return types (`React.FC`) for components, since that's apparently [discouraged](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components)?

Reply:
Cool, didn't know that, always nice to get rid of boilerplate 🙌 When did we decide that? Would be nice if there was a lint rule for it. There seem to be a codemod for it so could be nice to fix for all occurrences https://github.com/gndelia/codemod-replace-react-fc-typescript 
Maybe something for maintainance day :)

Reply:
I think that was a decision made in some PR discussion so probably not the best way to make decisions lol 🙈 

Great ideas, both adding linting rule and doing the codemod replacement on maintenance day 👌